### PR TITLE
fix: several skills-related runtime issues

### LINKED ISSUE #3674: Runtime skills configuration nesting mistake
The skills credential provider was attempting to extract `MicrosoftAppId` and `MicrosoftAppPassword` from the `runtimeSettings` -> `skills` scope in `appsettings.json`, whereas it should be using the root configuration scope.

### PR DESCRIPTION
**Pull request by joshgummersall**
Fixes #3674

### CODE REVIEW

**Review by mdrichardson — APPROVED** [review]

**Review by stevengum — COMMENTED** [review]
Unit tests? Otherwise LGTM

File: libraries/botbuilder-dialogs-adaptive-runtime-integration-express/src/index.ts [hunk 1]
```diff
@@ -5,16 +5,21 @@ import * as t from 'runtypes';
 import express from 'express';
 import path from 'path';
 import type { Server } from 'http';
-import { ActivityHandlerBase, BotFrameworkAdapter } from 'botbuilder';
+import type { ActivityHandlerBase, BotFrameworkAdapter, ChannelServiceRoutes } from 'botbuilder';
```
Comment (stevengum) [inline libraries/botbuilder-dialogs-adaptive-runtime-integration-express/src/index.ts:8]: Thanks! 👍🏼

File: libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts [hunk 2]
```diff
@@ -366,31 +366,33 @@ async function addSettingsBotComponents(services: ServiceCollection, configurati
...
-    let environment = configuration.string(['luis', 'environment']) ?? userName;
+    let environment = configuration.string(['luis', 'environment']) || userName;
     if (environment === 'Development') {
         environment = userName;
     }
 
     configuration.file(path.join(botRoot, 'generated', `luis.settings.${environment}.${luisRegion}.json`), true);
 
-    const qnaRegion = configuration.string(['qna', 'qnaRegion']) ?? 'westus';
+    const qnaRegion = configuration.string(['qna', 'qnaRegion']) || 'westus';
```
Comment (stevengum) [inline libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts:395]: Do we do the same defaulting in the .NET implementation? Is this `'westus'` region needed outside of the QnA Maker Knowledge Base authoring?

**Review by stevengum — CHANGES_REQUESTED** [review]

File: libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts (hunk 2 above)
Comment (joshgummersall) [reply libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts:395]: https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Extensions/ConfigurationBuilderExtensions.cs#L114

Comment (stevengum) [reply libraries/botbuilder-dialogs-adaptive-runtime/src/index.ts:395]: Thanks! Resolved.

**Review by stevengum — APPROVED** [review]
:shipit:

https://github.com/microsoft/botbuilder-js/issues/3677 filed for https://github.com/microsoft/botbuilder-js/pull/3675#pullrequestreview-658145428.

**Pull request merged**